_posts/2020-05-18-First-Trial-Data-Uploads.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: blog
+author: Tony Lorenzo
+---
+
+This post describes the first test trial of the Solar Forecast Arbiter
+framework, and a script that participants can uses to generate and post
+random forecasts for the trial.
```

Comment:
Sentence like "Read about the full testing plan in the [Trials Testing blog post](https://solarforecastarbiter.org/2020/02/12/Trials-Testing.html)"